### PR TITLE
Duration not being passed due to which error notification remains till infinite

### PR DESCRIPTION
**Pull request by avi123nash**
Adding a language or a theme? No

Duration was not being passed to error notification which is why it'll stay for infinite time and needs to be closed manually

If your change is visual (mainly themes) it would be extra awesome if you could include a screenshot.
error_notif

### CODE REVIEW

**Comment by avi123nash** [comment]
@Miodec Please review

**Comment by Miodec** [comment]
Im not exactly sure whats this meant to do

File: src/sass/style.scss [hunk 1]
```diff
@@ -710,7 +710,7 @@ a:hover {
   display: grid;
   gap: 1rem;
   position: fixed;
-  right: 1rem;
+  right: 12rem;
```
Comment (Miodec) [inline src/sass/style.scss:713]: Why this change?

File: src/js/leaderboards.js [hunk 2]
```diff
@@ -218,7 +218,7 @@ function updateLeaderboards() {
     })
     .catch((e) => {
       hideBackgroundLoader();
-      Notifications.add("Something went wrong: " + e.message, -1);
+      Notifications.add("Something went wrong: " + e.message, -1, 3);
```
Comment (Miodec) [inline src/js/leaderboards.js:221]: Errors are supposed to have no duration, so that they stay on screen.

**Review by Miodec — CHANGES_REQUESTED** [review]

File: src/sass/style.scss (hunk 1 above)
Comment (avi123nash) [reply src/sass/style.scss]: Reverted it back. I tested in local the error message was going out of screen but when I checked the actual website the error message was correct. My Bad

File: src/js/leaderboards.js (hunk 2 above)
Comment (avi123nash) [reply src/js/leaderboards.js:221]: Reverting this change too. Thanks

**Pull request merged**